src/components/Search/Form.js

```diff
@@ -0,0 +1,28 @@
+import { render } from '../../shared/dom';
+import { redirect } from '../../';
+import { routes } from '../../route';
+
+export const SearchForm = ({ renderOn, initSearch }) => {
+    const template = `
+        <form class="search-form" id="search-form">
```

Comment:
As I mentioned in task, it shouldn't be a form, we should react on user type, so we won't force user to press submit (we don't even have the submit button so it's bad UX when only "enter" button works), request should be send when user for example type 5 letters, and if he/she stops for like 0.5s, then we send the request